icinga2/synology-host.conf

```diff
@@ -0,0 +1,11 @@
+// `check_synology` example host configuration snippet for Icinga 2
+
+object Host "foo.example.org" {
+
+  // Configuration section for `check_synology`
```

Comment:
Should we use something more randomized as IP address?

Reply:
Sure we can do that. Do you have any suggestions?

Reply:
Do you suggest it would be better to use an IP address from another private IP range, like `10.123.456.789` or `172.16.1.2`?

Reply:
I've just added 885f845712399e to implement my suggestion. #24 will assist the user to figure things out easily, even without reading the documentation.

Reply:
> If not, I would prefer to use host names anyway. When doing so within documentation and configuration snippets, I usually use the [example.org domain](https://example.org/), which is exactly made for this purpose. So, what about synology.example.org, if that would make sense?

Exactly my thoughts